js/renovation/preact_wrapper/check_box.ts

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Most of the code is validation related and will be removed after [this card ](https://trello.com/c/bfMm4WY9)completion.